mathml/presentation-markup/too-few-children.html

```diff
@@ -0,0 +1,455 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Test if elements layout properly if they have too few children</title>
+<link rel="help" href="https://mathml-refresh.github.io/mathml-core/#script-and-limit-schemata">
```

Comment:
We should have links to the section corresponding to each element tested here. And these sections should clearly say how to handle "too few children". AFAIK, the core spec allows mrow-like elements with 0 children and the tabular section is not really well-explained (it defers to HTML).